desktop-wm/cwm/spec

```diff
@@ -0,0 +1,4 @@
+VER=7.1
+SRCS="tbl::https://github.com/leahneukirchen/cwm/archive/refs/tags/v$VER.tar.gz"
```

Comment:
I would recommend using `git::` source, as these tarballs (automatically generated by GitHub) can be overwritten unintentionally.